### ISSUE TITLE
Information to run the code

### ISSUE THREAD

**Issue by fahadayyaz44** [jensleitloff/CNN-Sentinel#10]
i want the classification of wheat using sentinel 2 imagery so it is able to classify images only from this data source? 
What should be least number tif maps to train the model?
How to save the model in the path you mentioned in classify images?
What are the basic codes only to classify sentinel two tif?

**Comment by fahadayyaz44** [comment]
Kindly reply

**Comment by felixriese** [comment]
> i want the classification of wheat using sentinel 2 imagery so it is able to classify images only from this data source?

Please clarify this question.

> What should be least number tif maps to train the model?

That depends on your images, your classification problem etc. There is no general answer.

> How to save the model in the path you mentioned in classify images?

See for example here: https://www.tensorflow.org/guide/keras/save_and_serialize

> What are the basic codes only to classify sentinel two tif?

To train a model, you can use [05_train_ms_from_scratch.py](https://github.com/jensleitloff/CNN-Sentinel/blob/master/py/05_train_ms_from_scratch.py).

- To classify your images, you can use [06_classify_image.py](https://github.com/jensleitloff/CNN-Sentinel/blob/master/py/06_classify_image.py).

**Comment by fahadayyaz44** [comment]
I have executed the code on google collab but its giving validation value
error from image filter rgb validation value is 500

**Comment by felixriese** [comment]
@fahadayyaz44 is that a question? How should we help you?

Overall, we cannot give you step-by-step help for your application. We only help on _specific_ questions, _constructive feedback_ and _bugs_.

**Comment by sanjivjha** [comment]
I am also interested in repurposing this code to do crop classification. If get training data for different crop classes, change class label definition, do you think any other part of the code will be impacted.

**Comment by jensleitloff** [comment]
Code should be fine, if you use the same folder structure. In general, you should use segmentation instead of classification. Furthermore, use time series with Sentinel data. This will significantly improve land use classification (especially results for crop type).
Take a look at this repository: https://github.com/MarcCoru/crop-type-mapping
